### ISSUE TITLE
How to contribute Chinese documents?

### ISSUE THREAD

**Issue by jiemojiemo** [google/googletest#3618]
**Does the feature exist in the most recent commit?**

No

**Why do we need this feature?**

Googletest is the most popular testing framework in C/C++, there are many Chinese coder use googletest in their project.  
But the learning curve is still curve for the Chinese beginners. So we need Chinese documents to help Chinese coder to use and learn googletest more effectively.

**Describe the proposal**

translate all docs to Chinese, and publish them to http://google.github.io/googletest

**Is the feature specific to an operating system, compiler, or build system version?**
No

**Comment by derekmauro** [comment]
I don't think we are going to be able to officially publish Chinese docs in our official repository. If you published them somewhere else, we would consider linking to them.

**Comment by Tomaw3y** [comment]
Hy idnt do anything cuz i have not idea how that s workSent from my T-Mobile 4G LTE Device
-------- Mensaje original --------De: Derek Mauro ***@***.***> Fecha: 23/2/22  11:55  (GMT-07:00) A: google/googletest ***@***.***> CC: Subscribed ***@***.***> Asunto: Re: [google/googletest] How to contribute Chinese documents? (Issue
  #3618) 
I don't think we are going to be able to officially publish Chinese docs in our official repository. If you published them somewhere else, we would consider linking to them.

—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/google/googletest/issues/3618#issuecomment-1049104528",
"url": "https://github.com/google/googletest/issues/3618#issuecomment-1049104528",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
